Resources/Locale/en-US/animals/udder/udder-system.ftl

```diff
@@ -5,3 +5,7 @@ udder-system-success = You fill {THE($target)} with {$amount}u from the udder.
 udder-system-dry = The udder is dry.
 
 udder-system-verb-milk = Milk
+
+udder-system-examine-overfed = This animal looks overfed
+udder-system-examine-hungry = This animal looks hungry
+udder-system-examine-none = This animal seems to not get hungry
```

Comment:
These might be nicer as `{ SUBJECT($entity) } looks overfed` (etc), passing in the entity when doing the Loc.GetString call.